deploy/helm/sumologic/values.yaml

```diff
@@ -1061,7 +1061,8 @@ fluent-bit:
   podLabels: {}
 
   ## Add custom pod annotations to fluent-bit daemonset pods
-  podAnnotations: {}
+  podAnnotations:
+    sumologic.com/otel: "true"
```

Comment:
This is a bit misleading, since it's not really about otel, but the name of the service. Maybe something like `sumologic.com/major-version: 3`.

We should also add a comment explaining why this exists.